src/main/include/subsystems/LimeLightUtil.h

```diff
@@ -52,7 +57,9 @@ namespace t34
 
         void Periodic();
 
-        inline void setTargetMode(TargetMode mode) { m_target_mode = mode; };
+        inline void SetTargetMode(TargetMode mode) { m_target_mode = mode; };
```

Comment:
What do you think about adding some explicit targeting methods?

```suggestion
        inline void SetTargetMode(TargetMode mode) { m_target_mode = mode; };
        inline void TargetAmp()     { SetTargetMode(TargetMode::kAmp); };
        inline void TargetSpeaker() { SetTargetMode(TargetMode::kSpeaker); };
        inline void TargetTrap()    { SetTargetMode(TargetMode::kTrap); };
```

The advantage is reducing the noise at the call site. For instance, when we change  the target mode in Robot.cpp, it currently looks like this:

```cpp
rc->limelight_util.SetTargetMode(t34::LimelightUtil::TargetMode::kSpeaker);
```

With the explicit targeting methods, it would look like this:

```cpp
rc->limelight_util.TargetSpeaker();
```

Obviously, that's a lot less cluttered.